prow/e2e-suite.sh

```diff
@@ -62,6 +62,7 @@ ISTIO_GO=$(cd $(dirname $0)/..; pwd)
 export HUB=${HUB:-"gcr.io/istio-testing"}
 export TAG="${GIT_SHA}"
 
+make init
```

Comment:
let's add init to where it's missing in the makefile 

```
ldemailly-macbookpro:istio ldemailly$ \rm -rf vendor/
ldemailly-macbookpro:istio ldemailly$ time make init
~/go/out ~/go/src/istio.io/istio
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp smb smbs smtp smtps telnet tftp 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 74.0M  100 74.0M    0     0  23.8M      0  0:00:03  0:00:03 --:--:-- 23.8M
~/go/src/istio.io/istio

real	0m21.812s
user	0m13.226s
sys	0m28.736s
ldemailly-macbookpro:istio ldemailly$ time make init
make: Nothing to be done for `init'.

real	0m0.096s
user	0m0.040s
sys	0m0.041s
```

Reply:
If this fixes the prow problem - we can do what you propose in a separate PR and evaluate what impact it has on the build time.